tests/test.py

```diff
@@ -25,7 +26,7 @@
 class CGTestCase(TestCase):
     @classmethod
     def setUpClass(cls):
-        cls.cg = Circleguard(KEY)
+        cls.cg = Circleguard(KEY, db_path='cache.db')
```

Comment:
I'll remove this, but is there a reason unit tests can't cache by default? Kinda annoying getting rate limited if you run the unit tests a couple times in a row (and this I think not everything is getting cached, so I still get rate limited even with this enabled).

Reply:
no reason at all - we should cache and just gitignore the db. Having that should stop any ratelimiting from happening though. I think the reason it's not is because we create a new `Circleguard` instance in most of the test cases. 

I think this is a holdover from when I was more worried about `Circleguard` saving state in some way and wanted to start fresh each time. That should no longer be a concern and the repeated instantiation can be removed

Reply:
any preferred path for the cache then?

Reply:
the tests directory - `cache_path = Path(__file__).parent`